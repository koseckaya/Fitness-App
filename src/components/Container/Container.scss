@import "src/styles/vars.scss";

.container {
  box-sizing: border-box;
  max-width: 1440px;
  padding: 0 15px;
  margin: 0 auto;
}
