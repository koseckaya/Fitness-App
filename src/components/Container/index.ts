export { default as Container } from './Container';
export type { Props as ContainerProps } from './Container';
