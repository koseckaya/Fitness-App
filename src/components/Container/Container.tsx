import { FC } from 'react';
import './Container.scss'

export type Props = {
  className?: string;
  children: React.ReactNode;
};

const Container: FC<Props> = ({ className='', children}: Props) => (
  <div
    className={`container ${className}`}>
      { children }
  </div>
);

export default Container;
