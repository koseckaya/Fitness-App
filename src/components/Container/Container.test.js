import React from 'react';
import renderer from 'react-test-renderer';
import Container from './Container';


it('renders correctly', () => {
    const tree = renderer
        .create(<Container></Container>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});