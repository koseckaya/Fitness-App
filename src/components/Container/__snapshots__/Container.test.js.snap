// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Container correctly 1`] = `
<div
  className="container "
/>
`;

exports[`renders correctly 1`] = `
<div
  className="container "
/>
`;
